sr_port/cache_stats.c

```diff
@@ -26,8 +26,8 @@ void cache_stats(void)
 	cache_entry	*csp;
 
 	total_attempts = cache_hits + cache_fails;
-	FPRINTF(stderr,"\nIndirect code cache performance -- Hits: %d, Fails: %d, Hit Ratio: %d%%\n",
-		cache_hits, cache_fails, total_attempts ? ((100 * cache_hits) / (cache_hits + cache_fails)) : 0);
+	FPRINTF(stderr,"\nIndirect code cache performance -- Hits: %d, Fails: %d, Hit Ratio: %lld%%\n",
```

Comment:
Since the hit ratio is going to be an integer value between 0 and 100, there is no need for a %lld. %d is good enough.